### DISCUSSION TITLE
How do I delete my CSS Battles account?

### DISCUSSION THREAD

**Discussion by markssematovics** [cssbattle/forum#89] [Bugs reports]
I had used my gitthub account for CSS Battles, but when re-entering the account on another device I used my e-mail accidentally creating a new CSSB account. Ofcourse the new account has no scores & is empty. But now that I have a direct e-mail account, the website won't let me enter the gitthub account any longer. 

Just saying :"You have already signed up with the same email using different social login. Eg. If it was Twitter previously, use Twitter now too."

So. How do I delete the new account, so I can access the gitthub one?

**Comment by chinchang** [comment]
Hey! Please DM us your email addresses with which accounts were made, on
***@***.*** or twitter (twitter.com/css_battle)

**Reply by markssematovics** [reply]
Github doesn't have DM features & I don't use twitter~

Wait but does this mean that users can't delete their CSS battles accounts themselves?

**Comment by PaulPR-44** [comment]
Any new solution to delete account in any other way then twitter dms ?
